src/assets/scss/App.scss

```diff
@@ -980,20 +980,71 @@
       padding: 8px;
       text-align: left;
       line-height: 18px;
-      transform: translate(0, -50%);
-      left: 30px;
-      top: 8px;
 
       &:before {
         content: "";
         position: absolute;
-        left: -8px;
         z-index: 1;
-        top: 50%;
-        margin-top: -6px;
-        border-top: 6px solid transparent;
-        border-bottom: 6px solid transparent;
-        border-right: 8px solid #000;
+      }
+
+      &.fs-tooltip-message--right {
```

Comment:
The one main file of `App.scss` for dealing with all the styles IMHO goes against the component and colocation practices. Since we've already made it this way, let us please try to refactor the places we touch and place the `style.scss` next to the component code itself.

So in this PR, we could do

```
components/
├─ Tooltip/
│  ├─ index.js
│  ├─ style.scss
```

Then `index.js` would import `style.scss` and any other atomic/app components relying on `components/Tooltip` would automatically import the relevant styles. Please let us consider it a mission to do this for all components we touch to make the code more modular.